_data/people.yml

```diff
@@ -78,7 +78,15 @@
     url: "bl"
     role: "Undergraduate Researcher"
     image: laufenberg.jpg
-
+    
+-group: 'Summer Interns'
```

Comment:
I was imagining that you could be listed among the other undergraduate students, but with the role as summer intern.  This works, too, and I'll leave it to your preference.

Reply:
I went ahead and changed it to Undergraduate students.